lib/jekyll/configuration.rb

```diff
@@ -175,7 +180,7 @@ def read_config_files(files)
       rescue ArgumentError => err
         Jekyll.logger.warn "WARNING:", "Error reading configuration. " \
                      "Using defaults (and options)."
-        $stderr.puts "#{err}"
+        $stderr.puts '#{err}'
```

Comment:
This won't work the way you think it works. Actually I don't know why it was like that in the first place.  `$stderr.puts err` is what that should be converted, actually, it's what it should have been in the first place.  I almost wanna do a blame to see if it was me who wrote that so I could slap myself.